src/ol_infrastructure/infrastructure/aws/opensearch/Pulumi.infrastructure.aws.opensearch.open.CI.yaml

```diff
@@ -0,0 +1,16 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-ci
+encryptedkey: AQICAHjs8ajWpT7YRhWXwI//wPkHX53RHlo0DjkgQOwCBTUBwQGxHicvhYUkUI1tZY6z5KO1AAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMKGV1jmWSqevJowLOAgEQgDtqpoHZ/9pM6v5phUMKeRFtsoKDS8J+M2/ysvlik23cJfDpefcuzm3ouwn0zzKhbijpx7JTC+xoCljJ2Q==
+config:
+  aws:region: us-east-1
+  consul:address: https://consul-apps-ci.odl.mit.edu
+  consul:scheme: https
+  environment:business_unit: operations
```

Comment:
This should be set to `mit-open` since the cluster is explicitly for that application (although it also powers OCW... :shrug: )